float64[] check_points
float64[] time
int8 mode
---
bool executable
string err_msg